Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.8.md

```diff
@@ -0,0 +1,106 @@
+12. Added new property named `AutoSize` to the `Button` control.  
+    GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+    * When set to `true`, the button's `Width` and `Height` will automatically be set based on the width and height of the button text.  When set to `false`, `Width` and `Height` are independent of the button text size.
+13. Added new method to the `IFont` interface and `Font` class that returns the bounds of each character of some given text relative to a position.  
+    GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+14. Added the ability to add a `Label` to a `Button`.  This can be done by sending in a `Label` instance into the `Button` constructor.  
+    GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+15. Added added additional constructors to the `Button` class to improve the ease of use when creating new buttons.  List of new constructors listed below.  
+    GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+    * `Button(Label label)`
+    * `Button(uint width, uint height)`
+    * `Button(uint width, uint height, Label label)`
+    * `Button(Point position, uint width, uint height)`
+    * `Button(Point position, uint width, uint height, Label label)`
+
+---
+
+<h2 style="font-weight:bold" align="center">Bug Fixes 🐛</h2>
+
+1. Fixed bug where animating and non-animating textures were not being rendered correctly.  
+   GitHub Issues: [#204](https://github.com/KinsonDigital/Velaptor/issues/204)
+2. Fixed an issue where a font size of 0 would crash the application  
+   GitHub Issues: [#189](https://github.com/KinsonDigital/Velaptor/issues/189)
+3. Fixed issue with sound time not displaying correctly in the sound testing scene of the testing application  
+   GitHub Issues: [#192](https://github.com/KinsonDigital/Velaptor/issues/192)
+4. Fixed an issue with the scroll mouse wheel not working.  
+   GitHub Issues: [#71](https://github.com/KinsonDigital/Velaptor/issues/71)
+
+---
+
+<h2 style="font-weight:bold" align="center">Breaking Changes 💣</h2>
+
+1. Changed the order of rendering from textures, text, then rectangle primitives to textures, rectangle primitives, then text.
+2. Changed the behavior of the `RectShape.BorderThickness` and `RectShape.CornerRadius` properties
+    * Previously, both properties would check the incoming value and restrict them to a limit of the smallest value between the half width or half height.  This was done to prevent any unintended rendering artifacts when rendering rectangles.  This behavior was removed from the `RectShape` and moved to the internal `RectGPUBuffer` class.  These checks and restrictions now occur right before sending the data to the GPU.
+3. Changed the `ControlBase.IsLoaded` property from `protected` to `private`
+    * The property was meant to be updated via the `ControlBase.LoadContent()` method
```

Comment:
Punctuation (add periods at end of sentences).  Check full document there are more punctuation issues.
```suggestion
3. Changed the `ControlBase.IsLoaded` property from `protected` to `private`.
    * The property was meant to be updated via the `ControlBase.LoadContent()` method.
```